### ISSUE TITLE
Improve performance load time on home page.

### ISSUE THREAD

**Issue by somecallmejosh** [somecallmejosh/nuxt-rest-countries#30]
Initial Lighthouse scores:

Performance: 83
Accessibility: 96 (missing lang attribute)
Best Practices: 100
SEO: 92 (no meta description)

FCP: 2.4s
LCP: 3.7s
TBT: 220ms
CLS: 0
SI: 2.5s
